lib/components/fileListItem/fileListItem.js

```diff
@@ -43,16 +50,26 @@ const FileListItem = ({
   );
 
   const icon = useMemo(() => {
-    if (loading) {
-      return <span className={styles.spinner} />;
+    if (loading || progress !== undefined) {
+      const progressProps = {
+        ...progressSettings,
+        className: cn(styles.progress, { [styles.infinite]: loading }),
+        percent: progress,
+      };
+
+      if (progress === undefined) {
+        progressProps.percent = 75;
```

Comment:
Nit: Is this just for visual effects? So there is some color on the bar? It might be worth adding a comment for why we set it when it is `undefined`.